apps/server/src/controllers/user.ts

```diff
@@ -241,3 +247,23 @@ export const changePassword = async (req: Request, res: Response) => {
     return res.status(500).json({ message: 'Error changing password' });
   }
 };
+
+export const saveKeybindings = async (req: Request, res: Response) => {
```

Comment:
you need to add this saveKeybindings function to the   routes/user.ts file 
```javascript
router.post('/:username/profile/keybinds', saveKeybindings);
``` 

With addition of this, I tested the save keybinds and it show success result.
This is my sample request http://localhost:5001/user/Suntoh/profile/keybinds
```
body:{
  "keybindings" :{
    "C": "s",
    "C#": "e",
    "D": "d",
    "D#": "r",
    "E": "f",
    "F": "g",
    "F#": "y",
    "G": "h",
    "G#": "u",
    "A": "j",
    "A#": "i",
    "B": "k"
  }
}
```

Reply:
or may be you can integrated it in the editUserProfile function if you don't want a separate post request